### PR TITLE
fix return type for GoogleGenerativeAIEmbeddings

### PR DESCRIPTION
**Pull request by adamvig96**
fixes https://github.com/langchain-ai/langchain-google/issues/247

### CODE REVIEW

File: libs/genai/langchain_google_genai/embeddings.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def embed_documents(
             )
         except Exception as e:
             raise GoogleGenerativeAIError(f"Error embedding content: {e}") from e
-        return [e.values for e in result.embeddings]
+        return [list(e.values) for e in result.embeddings]
```
Comment (lkuligin) [inline libs/genai/langchain_google_genai/embeddings.py:151]: do you mind also adding / fixing an integration test, please?

**Review by lkuligin — APPROVED** [review]

**Comment by lkuligin** [comment]
@adamvig96 could you fix small linter errors, and we're ready to merge! thanks a lot.

**Comment by adamvig96** [comment]
Done, thanks!

**Pull request merged**